contrib/board_files/beaglebone.conf

```diff
@@ -1,3 +1,6 @@
+[board]
+model = TI AM335x BeagleBone Black
```

Comment:
Ok, the different varients do have different pin outs to the headers, so this conf file should be named beaglebone_black.conf and it matching on the Black model specifically is correct. We can move it as a seperate patch.